webapp/src/translations/languages/en.json

```diff
@@ -627,7 +627,7 @@
         "createANewWallet": "Create a new wallet",
         "restoreWalletFromMnemonicSeed": "Restore wallet from recovery seed",
         "walletSyncNotice": "You will be able to create or restore a wallet when sync is complete.",
-        "loading": "Loading...",
+        "loading": "Loading ...",
```

Comment:
"Loading..." should remain as "Loading...", no `space`.

Reply:
You are right. I did some research and removed the space before the ellipsis.